### PR TITLE
271-refactor: Add width and height to header icons

### LINKED ISSUE #271: Lighthouse. Performance. Image elements do not have explicit width and height
## 📝 Describe the refactoring task

Image elements do not have explicit width and height

## 🛠️ Suggested improvement

[Link to explanation](https://web.dev/articles/optimize-cls?utm_source=lighthouse&utm_medium=devtools&hl=ru#images_without_dimensions)

## ℹ️ Additional context

Image

## ✅ Definition of Done (DoD):

Fix this Lighthouse issue.
Partly will be fixed in #439 

## 🕰️ Timeline:

Provide the deadline or estimated timeframe for the bug fix.

### PR DESCRIPTION
**Pull request by ansivgit**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [x] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

Added `width` and `height` to `dropdown-icon` in the `header` `menu-item`.

## Related Tickets & Documents

- Related Issue #271 
- Closes #271 

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Summary by CodeRabbit

- **Style**
	- Simplified the styling for the navigation component's label and improved image sizing within the dropdown, ensuring consistent dimensions and enhancing visual consistency across the interface.

### CODE REVIEW

**Review by thirdmadman — APPROVED** [review]

File: src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss [hunk 1]
```diff
@@ -52,6 +52,11 @@
       width: 16px;
       height: 17px;
```
Comment (Wystov) [inline src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss:53]: maybe make height here 16px too, as i can see it doesnt affect visually, but in code it looks like 'whats going on here?" when these magic numbers appear and differ from each other

File: src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss [hunk 2]
```diff
@@ -52,6 +52,11 @@
...
       margin-left: 10px;
+
+      > img {
```
Comment (Quiddlee) [inline src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss:56]: Do we really need `>` ?

Comment (ansivgit) [reply src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss:56]: No, but I like to be very specific about styles. In this case, it's the `img` that follows the `span`, not any `img` inside it.

File: src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss (hunk 1 above)
Comment (ansivgit) [reply src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss:53]: I remove it at all, thank you

**Review by Quiddlee — APPROVED** [review]

**Review by Wystov — APPROVED** [review]

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**